i18n/zh/docusaurus-plugin-content-docs/current/sys-utilities/gpshrink.md

```diff
@@ -0,0 +1,89 @@
+---
+title: pgshrink
+---
+
+# gpshrink
+
+Cloudberry Database 通过 gpshrink 系统工具缩容集群。集群资源空闲时，例如磁盘空间占用长期低于20%、CPU或内存占用率持续较低，则可以使用 gpshrink 来实现集群的缩容，从而节省服务器资源。用户可以通过 gpshrink 工具删除多余服务器上的 segment，从而实现集群缩容。
```

Comment:
Hey, please standardize the format for both Chinese and English letters, as well as numbers and letters. You can include spaces.